### PR TITLE
Add clarity text and an example of versioning eventTypes

### PR DESCRIPTION
**Pull request by duglin**
Just because @ac360 wanted to see it.

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -178,13 +178,16 @@ help intermediate gateways determine how to route the events.
...
+  The format of this is producer defined and might include information such
+  as the version of the `eventtype`.
 * Constraints:
    * REQUIRED
    * MUST be a non-empty string
    * SHOULD be prefixed with a reverse-DNS name. The prefixed domain dictates
             the organization which defines the semantics of this event type.
 * Examples
    * com.github.pull.create
+   * com.example.object.v2.delete
```
Comment (duglin) [inline spec.md]: `com.example.object.delete.v2`

**Comment by duglin** [comment]
@clemensv see if the text I added is headed in the direction you were thinking of on the call the other day.  Any suggested edits are welcome - from anyone.

**Comment by duglin** [comment]
@clemensv any comment on the latest edits?

**Comment by clemensv** [comment]
I like the guidance part. LGTM.

**Comment by duglin** [comment]
Approved on 11/29 call

**Pull request merged**